cri/ocicni/cni_manager.go

```diff
@@ -76,10 +77,14 @@ func (c *CniManager) SetUpPodNetwork(podNetwork *ocicni.PodNetwork) error {
 // TearDownPodNetwork is the method called before a pod's sandbox container will be deleted.
 func (c *CniManager) TearDownPodNetwork(podNetwork *ocicni.PodNetwork) error {
 	err := c.plugin.TearDownPod(*podNetwork)
```

Comment:
how about:
```
if err :=  c.plugin.TearDownPod(*podNetwork); err == nil {
    return nil
}
```

Reply:
`err` will be used later. So no need to change. :)

Reply:
make sense